src/Utilities/DataMappers.js

```diff
@@ -299,3 +300,35 @@ export const createCvesRows = (rows) => {
         ];
     }
 };
+
+export const createPatchSetRows = (rows, selectedRows = {}, filters) => {
+
+    const data =
+        rows &&
+        rows.map(row => {
+            return {
+                id: row.id,
+                key: row.id,
+                selected: selectedRows[row.id] !== undefined,
+                cells: [
+                    { title: row.name },
+                    { title: row.systems }
+                ]
+            };
+        });
+
+    return data.length > 0 && data ||
```

Comment:
According to statement above, `data` can possibly be `false` when rows are `false` or nullish, which can produce error when trying to access `length`, you probably want to add fallback above:
`data = rows ? rows.map(... : []`

Reply:
We use && and || on across Patch UI code. Because, we do not have to explicitly tell code to return undefined when we want to only check AND condition. 
E.G:
`const selectedCount = selectedRows && arrayFromObj(selectedRows).length;` [TableView: line 62](https://github.com/RedHatInsights/patchman-ui/blob/master/src/PresentationalComponents/TableView/TableView.js#L62)

`const advisoryType =  advisoryTypes.find(item => item.value === type) || advisoryTypes[3];` [AdvisoryType: line 7](https://github.com/RedHatInsights/patchman-ui/blob/master/src/PresentationalComponents/AdvisoryType/AdvisoryType.js#L7).